dadi/frontend/components/DocumentFilters/DocumentFilters.jsx

```diff
@@ -19,52 +19,75 @@ export default class DocumentFilters extends Component {
     } = this.props
 
     // Evaluate passed filter / store in state
+    let filters = (isValidJSON(filter) ? objectToArray(JSON.parse(filter), 'field') : []).map(this.deconstructFilters)
     this.state = {
```

Comment:
This is in pre-render. Just like https://github.com/dadi/publish/blob/07adcb74e9766f3fb1bc85894dec675bf4565121/dadi/frontend/components/FieldImage/FieldImage.jsx#L13-L16